src/task2/routs/routs.js

```diff
@@ -0,0 +1,31 @@
+import { persons } from '../exampleDB';
```

Comment:
misspeling in file name, `routes` correct name

Reply:
Corrected